### PR TITLE
Doc 9896 - Capella first getting started

### PR DESCRIPTION
**Pull request by maria-robobug**

### CODE REVIEW

**Comment by maria-robobug** [comment]
Just want to get this in before I publish Python.

File: modules/hello-world/examples/hello_world_t.py [hunk 1]
```diff
@@ -1,23 +1,41 @@
...
+    password,
+    # NOTE: If using SSL/TLS, add the certificate_path.
+    # We strongly reccomend this for production use.
+    # cert_path=cert_path
+)
 
-# needed to support SQL++ (N1QL) query
-from couchbase.cluster import QueryOptions
+# Connect options - global timeout opts
+timeout_opts = ClusterTimeoutOptions(kv_timeout=timedelta(seconds=10))
```
Comment (RichardSmedley) [inline modules/hello-world/examples/hello_world_t.py]: We need this on Capella, where the assumption is the client is on a laptop many hops from AWS,
but not for Server, which is likely to be local Docker.

Comment (RichardSmedley) [reply modules/hello-world/examples/hello_world_t.py]: (and WaitUntilReady is more useful for delays in spinning up the server and creating buckets locally)

File: modules/hello-world/pages/start-using-sdk.adoc [hunk 2]
```diff
@@ -188,118 +170,138 @@ Here are all of the import statements that you will need to run the sample code.
...
+[source,python]
 ----
-include::../examples/hello_world_t.py[tag=bucket]
+include::example$cloud.py[tag=bucket,indent=0]
 ----
 
-NOTE: We are working with the link:https://docs.couchbase.com/server/current/manage/manage-settings/install-sample-buckets.html[travel-sample] data bucket. If you are not, substitute _travel-sample_ with your _bucket-name_.
-
-If you are connecting to https://docs.couchbase.com/cloud/index.html[Couchbase Capella] rather than a local Couchbase Server, then also refer to the <<cloud-connections, Cloud section>>, below.
+NOTE: We are working with the link:https://docs.couchbase.com/server/current/manage/manage-settings/install-sample-buckets.html[travel-sample] data bucket.
```
Comment (RichardSmedley) [inline modules/hello-world/pages/start-using-sdk.adoc]: xref:server:manage:manage-settings/install-sample-buckets.adoc

**Review by RichardSmedley — APPROVED** [review]
Couple of small comments, otherwise LGTM :+1:

**Pull request merged**